lms/static/sass/features/_course-sock.scss

```diff
@@ -0,0 +1,172 @@
+.verification-sock {
+  display: inline-block;
+  position: relative;
+  width: 100%;
+  margin-top: $baseline;
+  max-width: 1180px;
```

Comment:
It is in the shared-v2 variables file, adding it to the partials-base file as well and using a variable.